definitions/mlops-machine_learning_model/golden_metrics.yml

```diff
@@ -2,6 +2,6 @@ totalInferences:
   title: "Total Inference"
   unit: REQUESTS_PER_MINUTE
   query:
-    select: uniqueCount(inference_id)
+    select: (uniqueCount(inference_id) / 30)
```

Comment:
@ykarlinski-newrelic why is that? Are you sure you are not selecting 30 minutes on purpose on the time picker box? It should sum the requests per minute on your query selected time box